apache/helpers.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class SslFileGenerationException extends Exception {}
+
+define('OUTPUT_DIR', '/etc/phpmyadmin/ssl');
```

Comment:
please re-use SSL_DIR

Reply:
Will be done in minutes

Reply:
Changed by PMA_SSL_DIR, as an ENV var setted in Dockerfile with default value as /etc/phpmyadmin/ssl